src/wallet/wallet2.cpp

```diff
@@ -1007,13 +1007,16 @@ void wallet2::set_unspent(size_t idx)
 //----------------------------------------------------------------------------------------------------
 void wallet2::check_acc_out_precomp(const tx_out &o, const crypto::key_derivation &derivation, const std::vector<crypto::key_derivation> &additional_derivations, size_t i, tx_scan_info_t &tx_scan_info) const
 {
+  hw::device &hwdev = m_account.get_device();
+  std::unique_lock<hw::device> hwdev_lock (hwdev);
```

Comment:
Please add `#include <mutex>`, otherwise it doesn't compile on Windows due to `std:: unique_lock` not being found.

Reply:
I'm not familiar at all about the difference between std and boost, but if `std::unique_lock` is to be avoided, @cslashm , please update the patch accordingly.

Reply:
I will fix that. I changed on other place but miss this one